assignments/E0.md

```diff
@@ -28,44 +28,44 @@ We would recommend adding at least an additional 20G of space, for a total of 50
 
 If you run out of disk space in the middle of compilation, you'll almost certainly have to
 delete files to free up enough space to resize your filesystem, which would likely
-mean running `make clean` and losing your progress in the compilation.
+mean running `$ make clean` and losing your progress in the compilation.
 
 0. First, install the requisite dependencies. On Fedora, you can use
 
-        sudo dnf builddep kernel
+        $ sudo dnf builddep kernel
```

Comment:
putting it here is good though

Reply:
Ok,
I have removed the `$ ` prefix from all cases that were inline with a paragraph from my previous commit, and amended it.
Should be good now.